source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -348,38 +351,19 @@ FilterDataStatus Filter::onData(ProcessorState& state, Buffer::Instance& data, b
       ENVOY_LOG(trace, "Call in progress for partial mode");
       state.setPaused(true);
       result = FilterDataStatus::StopIterationNoBuffer;
-    } else if (end_stream || state.queueOverHighLimit()) {
-      bool terminate;
-      std::tie(terminate, result) = sendStreamChunk(state, data, end_stream);
-
-      if (terminate) {
-        return result;
-      }
     } else {
-      // Keep on running and buffering
       state.enqueueStreamingChunk(data, false, false);
-
-      if (Runtime::runtimeFeatureEnabled(Runtime::defer_processing_backedup_streams) &&
```

Comment:
Do we no longer care about this runtime flag?

Reply:
In this case, it's more that the behavior that someone previously found was correct when this flag is enabled seems to be the correct behavior in all cases. I'm not actually sure who put that feature flag check in before, but when I looked at the code carefully to diagnose this issue I figured out that we didn't need the test because that was actually what we wanted to do all the time anyway.